src/models/item.js

```diff
@@ -0,0 +1,54 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const itemSchema = new Schema({
+  name: {
+    type: String,
+    maxLength: [30, "item name's max length is 30 characters"],
+    required: [true, 'Item name is required'],
+  },
+  type: {
+    type: String,
+    enum: {
+      values: [
+        'Books',
+        'Stationery',
+        'School Books',
+        'Novels',
+        'Test Books',
+        'Furniture',
+      ],
+      message:
+        "Item type should be equal to 'Books', 'Stationery', 'School Books', 'Novels', 'Test Books' or 'Furniture'",
+    },
+    required: [true, 'Item type is required'],
+  },
+  description: {
+    type: String,
+    required: [true, 'Description is required'],
+  },
+  photo: {
+    type: String,
+    required: [true, 'Item photo is required'],
+  },
+  state: {
+    type: Boolean,
+    default: true,
+  },
```

Comment:
What does `state` mean in context of item?

Reply:
is the item still available for lending or not

Reply:
In that case, it would be better to rename this field to something like `isAvailable` or change the type to `String` and use enum values "Available" and "Not available".
There should always be full clarity by looking at the schema what the field is going to be used for.

Reply:
Ok thanks
Changed it to `isAvailable`